spec/lib/web_of_science/map_names_spec.rb

```diff
@@ -58,6 +62,21 @@
     it_behaves_like 'contains_author_data'
   end
 
+  context 'MEDLINE records with anonymous names' do
+    let(:pub_hash_class) { described_class.new(medline_record_anon) }
+    let(:pub_hash) { pub_hash_class.pub_hash }
+
+    it 'works with WOS records with anonymous names' do
+      expect(pub_hash_class).to be_an described_class
```

Comment:
This test can never detect a "failure".  `AnyClass.new` must return an instance of the class or **Ruby** is broken.

Reply:
expect 1 == 1